vuepress/docs/next/tutorials/getting-started/gke-install.md

```diff
@@ -135,24 +139,24 @@ See the [Getting Started guide](../../docs/getting-started/README.md#login-to-en
 In order to scale an Entando Application across multiple nodes, you must provide a storage class that supports
 a `ReadWriteMany` access policy, e.g. by using a dedicated storage provider like GlusterFS.
 
-The example below uses the GCP Cloud Filestore to provide clustered storage. However, it is best practice to expose an existing enterprise clustered file solution as a StorageClass.
+The example below provides clustered storage via GCP Cloud Filestore. However, it is best practice to expose an existing clustered file solution as a StorageClass.
 
 ::: tip
-You do not need clustered storage to scale an Entando Application if you schedule all instances to the same node using a `ReadWriteOnce (RWO)` policy and taints on other nodes. Be aware of the impact to node resource allocation, as well as recovery if your application fails or becomes unreachable. Note that if the node fais or is shutdown, your application will be unresponsive while Kubernetes reschedules the pods to a different node.
+You do not need clustered storage to scale an Entando Application if you schedule all instances to the same node via taints on other nodes and a `ReadWriteOnce (RWO)` policy. Be aware of the impact to node resource allocation and to recovery, should your application fail or become unreachable. Note that if the node fails or is shutdown, your application will be unresponsive while Kubernetes reschedules the pods to a different node.
 :::
 
 ### Clustered Storage Using GCP Cloud Filestore
 1. In the left menu of the GCP portal, find the Storage section and select `Filestore` -> `Instances`
 
-2. Enable the Filestore if you haven't already
+2. Enable the Filestore, if you haven't already
 
 3. Select `Create Instance`
 
 4. Adjust the field values from the defaults as needed. Take note of your instance ID.
 
 5. Once the instance is created on the Filestore main page, note the IP address of your NFS
 
-6. Install the provisioner that creates the StorageClass to enable deployment of Entando Applications. Use the commands below, replacing YOUR-NFS-IP and YOUR-NFS-PATH with your instance ID and the IP address of your cluster.
+6. Install the provisioner that creates the StorageClass enabling deployment of Entando Applications. Use the commands below, replacing `YOUR-NFS-IP` and `YOUR-NFS-PATH` with your instance ID and the IP address of your cluster.
```

Comment:
add comma >> Install the provisioner that creates the StorageClass, enabling deployment of Entando Applications